spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/SpannerSparkTest.java

```diff
@@ -0,0 +1,138 @@
+package com.google.cloud.spark;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.spanner.Database;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.InstanceConfig;
+import com.google.cloud.spanner.InstanceId;
+import com.google.cloud.spanner.InstanceInfo;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spark.spanner.SpannerSpark;
+import com.google.cloud.spark.spanner.SpannerTable;
+import com.google.spanner.admin.database.v1.CreateDatabaseMetadata;
+import com.google.spanner.admin.instance.v1.CreateInstanceMetadata;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class SpannerSparkTest {
+
+  String databaseId = "spark-db";
+  String instanceId = "spanner-spark";
+  String projectId = "orijtech-161805";
```

Comment:
I'd pass the projectId, databaseId and instanceId as System env like SPANNER_EMULATOR_HOST.

Reply:
Done, thanks!